app/views/admin/items/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= bootstrap_form_for @item, :html => { multipart: true }  do |f| %>
+  <%= f.text_field :title %>
+  <%= f.text_area :description %>
+  <%= f.number_field :unit_price, label: "Price (in cents)"%>
```

Comment:
Yeah, the display name on the field is 'Price (in cents)', and it maps to unit_price in the background. I dont love that you have to enter it in cents right now, but was going to be a good amount of work to make it work in dollar format